tests/lib/file-finder.js

```diff
@@ -76,9 +76,11 @@ describe("FileFinder", () => {
                 finder = new FileFinder(["empty", uniqueFileName], process.cwd());
                 actual = finder.findAllInDirectoryAndParents(subdir);
```

Comment:
Nitpick: It might be easier to use `Array.from` here, which accepts an iterable and creates an array out of it.

```js
actual = Array.from(finder.findAllInDirectoryAndParents(subdir));
```

That way, the rest of the test won't need to be changed, so we can keep the assertions that were there previously (such as `actual.length === 2`).